tachyon/crypto/commitments/polynomial_openings_unittest.cc

```diff
@@ -198,39 +198,17 @@ TEST_F(PolynomialOpeningsTest, GroupByPolyOracleAndPoints) {
       PolyRef(&polys_[3]),
   };
 
-  if (grouped_poly_openings_vec[0].point_refs.size() == 2) {
-    EXPECT_EQ(grouped_poly_openings_vec[1].point_refs.size(), 1);
-    EXPECT_THAT(grouped_poly_openings_vec[0].point_refs,
-                testing::UnorderedElementsAreArray(expected_points_vec));
-    EXPECT_THAT(grouped_poly_openings_vec[1].point_refs,
-                testing::UnorderedElementsAreArray(expected_points_vec2));
-    for (const PolynomialOpenings<Poly>& poly_openings :
-         grouped_poly_openings_vec[0].poly_openings_vec) {
-      EXPECT_THAT(expected_polys_vec,
-                  testing::Contains(poly_openings.poly_oracle));
-    }
-    for (const PolynomialOpenings<Poly>& poly_openings :
-         grouped_poly_openings_vec[1].poly_openings_vec) {
-      EXPECT_THAT(expected_polys_vec2,
-                  testing::Contains(poly_openings.poly_oracle));
-    }
-  } else {
```

Comment:
Does this mean that this else statement never ran previously?

Reply:
For the first time we implemented `PolynomialOpenings`, it was flaky. Looking back, we should have thought weird.. but we accepted the changes..

Reply:
Ah I see - than maybe changing the title to "fix faulty logic from [#221](https://github.com/kroma-network/tachyon/pull/211)" might be better. I got confused thinking the commit was a new test feature or so.

Reply:
Actually #221 fixes the bug so missing changes looks more correct to me.